src/components/income/F1099Info.tsx

```diff
@@ -6,7 +6,7 @@ import { Actions, add1099, remove1099 } from '../../redux/actions'
 import { PagedFormProps } from '../pager'
 import { TaxesState, Person, PersonRole, Supported1099, Income1099Type } from '../../redux/data'
 import DeleteIcon from '@material-ui/icons/Delete'
-import { GenericLabeledDropdown, LabeledInput } from '../input'
+import { Currency, GenericLabeledDropdown, LabeledInput } from '../input'
 import { Patterns } from '../Patterns'
 
 const showIncome = (a: Supported1099): ReactElement => {
```

Comment:
1099 DIV and 1099 INT income should be formatted using the currency component as well as 1099 B